pkg/sfu/buffer/helpers.go

```diff
@@ -275,3 +276,38 @@ func IsH264Keyframe(payload []byte) bool {
 	}
 	return false
 }
+
```

Comment:
moving this from `pkg/sfu/helpers.go` to here. There was an NTPTime stuff in relay. Putting it in a common place for re-use.